cartridges/int_stickyio_sfra/cartridge/scripts/subscription/subscriptionHelpers.js

```diff
@@ -153,6 +152,7 @@ function getSubscriptions(currentCustomer, querystring, locale, billingModels) {
         var thisOrder = OrderMgr.getOrder(orders[i].orderNumber, orders[i].orderToken);
         customerID = thisOrder.getCustomer().ID; // this gets overwritten every time, but that's ok
         var orderShipments = thisOrder.getShipments();
+        orderModel = orders[i];
```

Comment:
should we change all the orders[i] refs below to use orderModel?

Reply:
This wasn't something i modified as part of SFCC-75.  It probably could be changed without issue.